templates/admin/mentoring/index.html.twig

```diff
@@ -19,18 +17,20 @@
         </thead>
         <tbody>
         {% for mentoring in mentorings %}
-        <tr>
-            <th scope="row">{{ mentoring.id }}</th>
-            <td>{{ mentoring.student.user.firstname }}</td>
-            <td>{{ mentoring.student.user.lastname }}</td>
-            <td>{{ mentoring.mentor.user.firstname }}</td>
-            <td>{{ mentoring.mentor.user.lastname }}</td>
-            {% if mentoring.isAccepted == true %}
-                <td><a class="btn btn-danger" href="{{ path('admin_mentoring_desactivate', {'id': mentoring.id}) }}) }}">Désactiver le mentorat</a></td>
-            {% else %}
-            <td><a class="btn btn-warning" href="#">Le mentorat est désactivé</a></td>
-            {% endif %}
-        </tr>
+        {% if mentoring.student != null %}
+            <tr>
+                <th scope="row">{{ mentoring.id }}</th>
+                <td>{{ mentoring.student.user.firstname }}</td>
+                <td>{{ mentoring.student.user.lastname }}</td>
+                <td>{{ mentoring.mentor.user.firstname }}</td>
+                <td>{{ mentoring.mentor.user.lastname }}</td>
+                {% if mentoring.isAccepted == true %}
```

Comment:
same thing here, in twig we don't usee == for testing : https://twig.symfony.com/doc/2.x/tests/sameas.html